openstack/cerebro/cerebro-main/values.yaml

```diff
@@ -0,0 +1,20 @@
+image:
+  pullPolicy: IfNotPresent
+  tag: SET-AS-HELM-PARAMETER # This should be set via --set to a git commit hash
+  registry: SET-AS-HELM-PARAMETER
+
+vernemq:
+  vernemq:
+    image:
+      repository: SET-AS-HELM-PARAMETER
+      tag: SET-AS-HELM-PARAMETER  # This should be set via --set to a git commit hash
+
+
+migrationRecommenderService:
+  prometheusClientPort: 8092

```

Comment:
These ports should be greater than `9000` to match the prometheus exporter pattern.

Reply:
Changed `8092` to `9092` for all three microservices.